lib/common/nodes.c

```diff
@@ -9,8 +9,23 @@
 
 #include <crm_internal.h>
 
+#include <libxml/tree.h>        // xmlNode
 #include <crm/common/nvpair.h>
 
+/*!
+ * \internal
+ * \brief Check whether a node is online
+ *
+ * \param[in] node  Node to check
+ *
+ * \return true if \p node is online, otherwise \false
```

Comment:
either `\c false` or `false` but not `\false`

Reply:
argh